crates/common/tedge_config/src/tedge_config_cli/tedge_config_repository.rs

```diff
@@ -8,48 +8,17 @@ use super::figment::ConfigSources;
 use super::figment::FileAndEnvironment;
 use super::figment::FileOnly;
 use super::figment::UnusedValueWarnings;
-use super::new;
 
 /// TEdgeConfigRepository is responsible for loading and storing TEdgeConfig entities.
 #[derive(Debug, Clone)]
 pub struct TEdgeConfigRepository {
     config_location: TEdgeConfigLocation,
```

Comment:
Given how many times the `load()` method is called (24 times in 23 files), we probably have multiple copies of `TEdgeConfig` per process. We could lazily initialize it and only return a reference to the single instance. The struct is not very large, so it's not a big problem, but perhaps this will be easier to do sooner rather than later.

Reply:
> Given how many times the `load()` method is called (24 times in 23 files), we probably have multiple copies of `TEdgeConfig` per process. We could lazily initialize it and only return a reference to the single instance. The struct is not very large, so it's not a big problem, but perhaps this will be easier to do sooner rather than later.

Added to https://github.com/thin-edge/thin-edge.io/issues/2153